packages/dashboard/package.json

```diff
@@ -62,6 +62,7 @@
     "react-use": "^17.4.0",
     "unstated-next": "^1.1.0",
     "urlcat": "^3.1.0",
+    "immer": "^10.0.1",
```

Comment:
We seem to have no need of a new library.

Reply:
we already uses immer so it's not a new library.